retrofit/src/main/java/retrofit2/http/Field.java

```diff
@@ -30,21 +30,22 @@
  * field pair for each non-{@code null} item.
  * <p>
  * Simple Example:
- * <pre>
+ * <pre>{@code
```

Comment:
Sounds good to me. That format was obtained by Kevin B at one point by trial & error; things might work better with the latest Javadoc.